### PR TITLE
Feature: Show exif data in NowPlaying overlay

### PR DESCRIPTION
**Pull request by wutschel**
## Description

This PR adds the support for displaying EXIF data in the NowPlaying overlay screen. It will show the basic information for picture, minimum:
- image resolution
- image format (e.g. jpeg)

When EXIF data is present there is a selection presented which I understood as most relevant from browsing several photography related internet sites:

- "Slideshow.Resolution"
- "Slideshow.Filename"
- "Slideshow.CameraModel"
- "Slideshow.EXIFTime"
- "Slideshow.Aperture"
- "Slideshow.ISOEquivalence"
- "Slideshow.ExposureTime"
- "Slideshow.Exposure"
- "Slideshow.ExposureBias"
- "Slideshow.MeteringMode"
- "Slideshow.FocalLength"

In addition the readability of the the overlay content is improved in three areas: 
- font size for description scales up with increasing display size
- description text is now block aligned
- updated aspect ratio background icon does not interfere if picture resolution or aspect ratio text needs more space

Screenshots:
https://abload.de/img/bildschirmfoto2023-056lcyt.png (iPhone)
https://abload.de/img/bildschirmfoto2023-05gscxe.png (iPad)

## Summary for release notes

Feature: Show exif data in NowPlaying overlay
Improvement: Better readability for NowPlaying overlay

### CODE REVIEW

File: XBMC Remote/NowPlaying.m [hunk 1]
```diff
@@ -941,6 +950,47 @@ - (void)loadCodecView {
...
+                     [infoString appendAttributedString:[self formatInfo:@"Resolution" text:methodResult[@"Slideshow.Resolution"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Aperture" text:methodResult[@"Slideshow.Aperture"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Exposure Time" text:methodResult[@"Slideshow.ExposureTime"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Exposure Mode" text:methodResult[@"Slideshow.Exposure"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Exposure Bias" text:methodResult[@"Slideshow.ExposureBias"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Metering Mode" text:methodResult[@"Slideshow.MeteringMode"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Focal Length" text:methodResult[@"Slideshow.FocalLength"]]];
+                     [infoString appendAttributedString:[self formatInfo:@"Camera Model" text:methodResult[@"Slideshow.CameraModel"]]];
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:989]: why are these strings not localized?

File: XBMC Remote/NowPlaying.m [hunk 2]
```diff
@@ -941,6 +950,47 @@ - (void)loadCodecView {
...
+                     [infoString appendAttributedString:newLine];
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:990]: looks like you have double line break after the last element, is this intended?

File: XBMC Remote/NowPlaying.m [hunk 3]
```diff
@@ -954,6 +1005,27 @@ - (void)loadCodecView {
     }];
 }
 
+- (NSAttributedString*)formatInfo:(NSString*)name text:(NSString*)text {
+    int fontSize = descriptionFontSize;
+    // Bold and gray for label
+    name = [NSString stringWithFormat:@"%@: ", name];
+    NSDictionary *boldFontAttrib = @{
+        NSFontAttributeName: [UIFont boldSystemFontOfSize:fontSize],
+        NSForegroundColorAttributeName: UIColor.lightGrayColor
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:1014]:
```suggestion
        NSForegroundColorAttributeName: UIColor.lightGrayColor,
```

File: XBMC Remote/NowPlaying.m [hunk 4]
```diff
@@ -954,6 +1005,27 @@ - (void)loadCodecView {
...
+    };
+    // Normal and white for the text
+    NSMutableAttributedString *string1 = [[NSMutableAttributedString alloc] initWithString:name attributes:boldFontAttrib];
+    text = [NSString stringWithFormat:@"%@\n", text];
+    NSDictionary *normalFontAttrib = @{
+        NSFontAttributeName: [UIFont systemFontOfSize:fontSize],
+        NSForegroundColorAttributeName: UIColor.whiteColor
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:1021]:
```suggestion
        NSForegroundColorAttributeName: UIColor.whiteColor,
```

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: XBMC Remote/NowPlaying.m (hunk 3 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m:1014]: Nice one, this was a copy-paste issue. Will add a commit to improve this for the origin as well.

File: XBMC Remote/NowPlaying.m (hunk 4 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m:1021]: Same as above.

File: XBMC Remote/NowPlaying.m (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m:990]: Thanks, obsolete remnant of intermediate versions. Will remove this.

File: XBMC Remote/NowPlaying.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m:989]: I had seen these as kind of image attributes and therefore kept this in technical english. But I now see Kodi server has translation for those. Will add localization as well then.

**Comment by wutschel** [comment]
Updated with several fixups, will rebase once approved.

**Review by kambala-decapitator — APPROVED** [review]

**Comment by wutschel** [comment]
Squashed and rebased to master.

**Pull request merged**